README.md

```diff
@@ -32,7 +32,7 @@ Paste your Trello API key and token from [here](https://trello.com/app-key) into
 Identify the id of the board you wish to connect with, as well as the list ids of the to-do, doing, and done lists and paste them in.
```

Comment:
This line needs updating. The requirement is now that the board has the correct list names.